core/src/main/java/org/mskcc/cbio/portal/dao/DaoCancerStudy.java

```diff
@@ -330,7 +327,10 @@ public static void addCancerStudy(CancerStudy cancerStudy, boolean overwrite) th
                 pstmt.setString(8, StringUtils.join(groups, ";"));
             }
             pstmt.setString(9, cancerStudy.getShortName());
-
+          	//status is UNAVAILABLE until other data is loaded for this study. Once all is loaded, the 
+            //data loading process can set this to AVAILABLE:
+            //TODO - use this field in parts of the system that build up the list of studies to display in home page:
+            pstmt.setInt(10, Status.UNAVAILABLE.ordinal());
```

Comment:
We could do that. Do you want me to disable the part where we set it to AVAILABLE at the end as well? However, we would like to implement the missing parts for this *soon* as the current import process is not transactional (this means that whenever there is a problem and the process stops - which is the key requirement of #848 for which this PR was made - you end up with a corrupted study in the study list). Ideally we don't display such studies in the UI, to avoid unexpected issues when users start querying half loaded studies. I believe @priti88 and @ecerami would be interested in this feature in the context of their requirements for #848.